src/index_spec.coffee

```diff
@@ -167,3 +167,39 @@ describe 'alphabetize_keys', ->
           errors = coffeelint.lint notAlphabetical[name]
           expect(errors).to.have.lengthOf 2
           expect(errors[0].rule).to.eql 'alphabetize_keys'
+
+  context 'overrides', ->
+    beforeEach ->
+      @str = '''
+        class A
+          methodD: ->
+          methodB: ->
+          methodA: ->
+          methodC: ->
+        '''
+
+    context 'matches overrides ordering', ->
+      it 'returns no errors', ->
+        opts = alphabetize_keys: overrides: ['methodD', 'methodB', 'methodA', 'methodC']
+        errors = coffeelint.lint @str, opts
+        expect(errors).to.be.empty
+
+    context 'does not match overrides ordering', ->
+      it 'returns an error', ->
+        opts = alphabetize_keys: overrides: ['methodD', 'methodA', 'methodB', 'methodC']
+        errors = coffeelint.lint @str, opts
+        expect(errors).to.have.lengthOf 1
+        expect(errors[0].rule).to.eql 'alphabetize_keys'
+
+    context 'with other keys', ->
+      it 'alphabetical', ->
+        opts = alphabetize_keys: overrides: ['methodD', 'methodB']
+        errors = coffeelint.lint @str, opts
+        expect(errors).to.be.empty
+
+      it 'not alphabetical', ->
+        opts = alphabetize_keys: overrides: ['methodB', 'methodA']
```

Comment:
This one I'm not sure about. If `methodD` and `methodC` were switched, it wouldn't error because `methodB` and `methodA` are added to the different list and thus not considered when checking the order of the none override methods. So that means that methods could exist before the group that should be alphabetically after? Does that make sense?